demos/JWST/S5_fit_par_template.epf

```diff
@@ -70,6 +70,25 @@ u2          0.84            'free'          0           1           U
 # osc_t0          0.3163  'fixed'         0            1              U
 # osc_t1          0.3163  'free'          0.3          0.4            U
 
+# ------------------
+# ** Star Spot parameters **
```

Comment:
Please also add these changes to the `docs/media/S5_fit_par_template.epf` file which is shown on the https://eurekadocs.readthedocs.io/en/latest/ecf.html page

Reply:
done